be/src/common/config.h

```diff
@@ -1195,4 +1195,7 @@ CONF_mInt64(pk_dump_interval_seconds, "3600"); // 1 hour
 
 // whether enable query profile for queries initiated by spark or flink
 CONF_mBool(enable_profile_for_external_plan, "false");
+
+// jit LRU cache size, will div 32 shards, i.e., 100 per shard by default, taking more memory if larger
```

Comment:
Could you comment on how much memory it takes when it is set to default?

Reply:
it depends on complex expressions, we will test memory usage later. As commented, we will evit from cache by the memory usage in the next work, I don't find a better way to collect memory usage now.

Reply:
yes, I will fix the memory issue soon after passing the CI.